packages/block-library/src/image/image.js

```diff
@@ -544,13 +540,9 @@ export default function Image( {
 		// With the current implementation of ResizableBox, an image needs an
 		// explicit pixel value for the max-width. In absence of being able to
 		// set the content-width, this max-width is currently dictated by the
-		// vanilla editor style. The following variable adds a buffer to this
-		// vanilla style, so 3rd party themes have some wiggleroom. This does,
-		// in most cases, allow you to scale the image beyond the width of the
-		// main column, though not infinitely.
-		// @todo It would be good to revisit this once a content-width variable
-		// becomes available.
-		const maxWidthBuffer = maxWidth * 2.5;
+		// vanilla editor style. We'll use the clientWidth here, to prevent the width
+		// of the image growing larger than the width of the block column.
+		const maxWidthBuffer = clientWidth;
```

Comment:
Should we compare with `maxWidth` here and choose the minimum?

```suggestion
		const maxWidthBuffer = Math.min( clientWidth, maxWidth * 2.5 );
```

Reply:
@kevin940726 I believe the goal of this commit is to prevent the image from being larger than its container (`clientWidth`). Is there an instance where `maxWidth * 2.5` is smaller than the container?

Reply:
I'm not sure! It's a question and I'm not very familiar with the `maxWidth` settings. I think it's fine to keep just the `clientWidth` though.